### DISCUSSION TITLE
The meaning of the control point coordinates of the Bezier curve

### DISCUSSION THREAD

**Discussion by ChallengerDefier** [openTCS/opentcs#13] [Q&A] [answered]
I have drawn a bezier curve between two points, but the coordinates of the control points seem to be too different from the coordinates of the points.
How are the coordinates of the control points of the Bezier curve defined? Isn't it layout coordinates?

**Comment by martingr** [comment, marked as answer]
Hi,

the coordinates of the control points refer to the non-inverted, non-scaled coordinate system. 

In the coordinate system in openTCS, the Y-axis is inverted so that its positive direction points upwards. In addition, the x and y axes can be scaled via the layout element in the plant model. In a new plant model, the default scaling for both axes is 50.0 mm.

If the scaling of both axes is set to 1.0 mm, you can see that the coordinates of the control points fit the coordinate system. Only the y-coordinates are inverted then.

Best regards,
Martin

**Reply by ChallengerDefier** [reply]
Indeed, thank you very much for your answer.
